docs/sync.md

```diff
@@ -11,19 +11,32 @@ The two steps are:
 
 2. On the live website at [https://prairielearn.engr.illinois.edu](https://prairielearn.engr.illinois.edu), go to your course, select the "Sync" page in the menu bar, and then click the "Pull from remote git repository” button.
 
-## Sync permissions
+## Course admin permissions
```

Comment:
Changing headers changes the link targets. There is one that needs to be updated near the top of `accessControl.md`